src/index.js

```diff
@@ -4,6 +4,9 @@
  * Module dependencies.
  */
 
+const {
+  strategies: { REDACT, TRIM, TRIM_AND_LIST }
+} = require('./enums');
```

Comment:
```suggestion
const { strategies } = require('./enums');

const { REDACT, TRIM, TRIM_AND_LIST } = strategies.
```

Deconstruct in the constants section.

Reply:
Done